### PR TITLE
Support new wreck in mocks

### PR DESCRIPTION
**Pull request by geek**

### CODE REVIEW

File: test/mock.js [hunk 1]
```diff
@@ -269,6 +269,6 @@ exports.override = function (uri, payload) {
 
 exports.clear = function (uri) {
 
-    Wreck.get = internals.nipple.get;
-    Wreck.post = internals.nipple.post;
+    Wreck.get = internals.wreck.get;
+    Wreck.post = internals.wreck.post;
 };
```
Comment (lloydbenson) [inline test/mock.js]: Missing EOL

Comment (lloydbenson) [inline test/mock.js]: Lots of failed tests.  I am guessing this may be broke without this PR.

Comment (geek) [inline test/mock.js]: EOL exists, ya... wanted to see if travis failed too.  Something is broke since last publish of bell

**Comment by lloydbenson** [comment]
Looks good!

**Pull request merged**